Doc/howto/functional.rst

```diff
@@ -65,11 +65,10 @@ output must only depend on its input.
 
 Some languages are very strict about purity and don't even have assignment
 statements such as ``a=3`` or ``c = a + b``, but it's difficult to avoid all
-side effects.  Printing to the screen or writing to a disk file are side
-effects, for example.  For example, in Python a call to the :func:`print` or
-:func:`time.sleep` function both return no useful value; they're only called for
-their side effects of sending some text to the screen or pausing execution for a
-second.
+side effects - such as printing to the screen or writing to a disk file. Another
+example - a call to the :func:`print` or :func:`time.sleep` function both return
```

Comment:
I don't think there needs to be as much change here as is currently in this PR.  I like the combination of the first two sentences, but without the added ` - `.  I don't think anything needs to change past the newly-combined first sentence.

Reply:
Thanks for your feedback @zware. I've provided new changes